org.librarysimplified.r2.demo/src/main/java/org/librarysimplified/r2/demo/DemoActivity.kt

```diff
@@ -30,35 +43,35 @@ class DemoActivity : AppCompatActivity(), DemoNavigationControllerType, SR2Contr
       thread
     })
 
+  private var epubFile: File? = null
+  private var controller: SR2ControllerType? = null
+  private var controllerSubscription: Disposable? = null
+
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
 
-    this.setContentView(R.layout.demo_host)
-
     if (savedInstanceState == null) {
-      val fragment = DemoIntroFragment()
-      this.supportFragmentManager.beginTransaction()
-        .replace(R.id.mainFragmentHost, fragment, "MAIN")
-        .commit()
+      setContentView(R.layout.demo_activity)
+
+      val browseButton = findViewById<Button>(R.id.browse_button)!!
```

Comment:
Minor point: I prefer to use explicit `this`. It means you never have to guess whether a reference refers to a local variable/function, field, or static constant.

Reply:
Ah, yes, this is just a habit I've developed from other projects. Happy to stick with explicit this since it's an established convention!

We should codify these styles in a document somewhere.

Reply:
I'm gonna merge this in as-is for now, since the style is at least consistent in that file, but will fix this in a follow-up.